crates/packaging/src/https.rs

```diff
@@ -29,13 +30,23 @@ pub struct PackageMetadata<'a> {
 /// - .tar.br
 const VALID_EXTENSION_SUFFIXES: [&str; 2] = [".gz", ".br"];
 
-#[derive(Debug)]
+/// Characters that could be misleading if present in URLs:
+///
+/// @ - For now we avoid usage of the @, to avoid the "tld zip" attack vector
+/// ⁄ - U+2044 Fraction Slash
+/// ∕ - U+2215 Division Slash
+/// ／ - U+FF0F Fullwidth Solidus
+/// ⧸ - U+29F8 Big Solidus
+const MISLEADING_CHARACTERS_IN_URL: [&str; 5] = ["@", "\u{2044}", "\u{2215}", "\u{FF0F}", "\u{29F8}"];
```

Comment:
🎨 Minor note, but I think it'd be easier to read with the comments inline after each of the strings in the array, instead of in a separate comment above

Reply:
Also, I think these can probably be `char`s instead of `&str`s (so, single quotes instead of double quotes) - that would save some memory.

Reply:
The `char` idea worked like a .. *char*m ☜(ﾟヮﾟ☜) 
Thanks for the tip 🙏